src/gui/owncloudgui.cpp

```diff
@@ -89,6 +89,16 @@ ownCloudGui::ownCloudGui(Application *parent)
 
     connect(_tray.data(), &Systray::shutdown,
         this, &ownCloudGui::slotShutdown);
+    connect(_tray.data(), &Systray::openShareDialog,
+        this, [=](const QString &sharePath, const QString &localPath, const bool publicLink = false)
+            {
```

Comment:
I'd move the opening curly brace at the end of the previous line

Reply:
Same thing as with the this->; There is no consistent style for this throughout the code base. Guess I'll adapt yours for this ✔️

Reply:
Well, interestingly we seem to be consistent on the opening curly brace placement. We're totally not consistent on space after the capture or not or space before the opening curly brace or not.